docs/standards/smart-contracts/lsp7-digital-asset.md

```diff
@@ -13,7 +13,7 @@ sidebar_position: 8
 
 The **LSP7DigitalAsset** contract represents digital assets for either fungible or non-fungible tokens where minting and transferring are specified with an amount of tokens. It has some functions from **[ERC20](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC20/ERC20.sol)** and **[ERC777](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC777/ERC777.sol)** with more upgraded features.
 
-This contract serves as a **Fungible Token Contract** when `isNFT` bool is set to **false** in the `constructor(...)` and otherwise serves as a **Non-Fungible Token Contract**.
+This contract serves as a **Fungible Token Contract** when `isNonDivivsible` bool is set to **false** in the `constructor(...)` and otherwise serves as a **Non-Fungible Token Contract**.
```

Comment:
```suggestion
This contract serves as a **Fungible Token Contract** when `isNonDivisible` bool is set to **false** in the `constructor(...)` and otherwise serves as a **Non-Fungible Token Contract**.
```